framework/src/araui/include/ui_animation_internal.h

```diff
@@ -0,0 +1,87 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __UI_ANIMATION_INTERNAL_H__
+#define __UI_ANIMATION_INTERNAL_H__
+
+#include <tinyara/config.h>
+#include <stdint.h>
+#include "libs/vec.h"
```

Comment:
vec is moved to the external.
need to change <vec/vec.h>

Reply:
@gcjjyy fixed